kafka-ui-react-app/src/components/Connect/List/__tests__/List.spec.tsx

```diff
@@ -84,39 +84,4 @@ describe('Connectors List', () => {
       ).toBeInTheDocument();
     });
   });
-
```

Comment:
Hey , why are we deleting this ? just curios can you elaborate please

Reply:
@Mgrdich, I moved this test case into `ListAction.spec.ts`. Maybe it's more visual